azkaban-webserver/src/main/java/azkaban/webapp/servlet/ExecutorServlet.java

```diff
@@ -338,6 +340,24 @@ private void handleExecutionFlowPage(HttpServletRequest req,
       page.render();
       return;
     }
+    
+    Props props = getApplication().getServerProps(); 
+    String execExternalLinkURL = 
+        props.getString("execution.external.link.url", "");
+    String execExternalLinkLabel = 
+        props.getString("execution.external.link.label", "External Analyzer");
+        
+    if (StringUtils.isNotEmpty(execExternalLinkURL)) {
+      StringBuilder builder = new StringBuilder();
+      builder.append(req.getRequestURL());
+      builder.append("?");
+      builder.append(req.getQueryString());
+      String flowExecutionURL = builder.toString();
```

Comment:
Could you help me understand why Dr. Elephant needs the full url instead of just the execution id? How difficult is it to add the capability? 

The URL would look cleaner with just the execution id.